web/src/beta/components/SidePanelSectionField/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState, ReactNode } from "react";
+
+import { styled, useTheme, PublishTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+const SidePanelSectionField: React.FC<{
+  publishedTheme?: PublishTheme;
+  title: string;
+  children?: ReactNode;
+}> = ({ publishedTheme, title, children }) => {
+  const theme = useTheme();
+  const [opened, setOpened] = useState(false);
+
+  return (
+    <Field>
+      <Header onClick={() => setOpened(!opened)}>
+        <Text size="xs" color={theme.other.white} otherProperties={{ height: "16px" }}>
+          {title}
+        </Text>
+        <Icon
+          icon="arrowToggle"
+          size={12}
+          color={publishedTheme?.mainText || theme.main.text}
+          style={{ rotate: opened ? "90deg" : "180deg" }}
+        />
+      </Header>
+      {opened && children}
+    </Field>
+  );
+};
+
+const Field = styled.div`
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
```

Comment:
Do you need this? This component seems that display: block is enough. ( `div` default behaviour is `block` )
Please check `width: 100%` as well

Reply:
(note) did not check modification yet, need to check on local env (?)